df_auth/social_auth.py

```diff
@@ -0,0 +1,24 @@
+from django.conf import settings
+from social_core.utils import get_strategy
+from social_django.strategy import DjangoStrategy
+from social_django.utils import psa
+from social_django.utils import STORAGE
+
+
+class DRFStrategy(DjangoStrategy):
+    def __init__(self, storage, request=None, tpl=None):
+        self.request = request
+        super(DjangoStrategy, self).__init__(storage, tpl)
```

Comment:
this is not pythonic 3 super should be without class

Reply:
Done.👌